src/librbd/AioImageRequest.cc

```diff
@@ -335,7 +335,7 @@ void AioImageRead<I>::send_request() {
                                                                     req);
         image_ctx.aio_read_from_cache(extent.oid, extent.objectno,
                                       &req->data(), extent.length,
-                                      extent.offset, cache_comp, m_op_flags);
+                                      extent.offset, cache_comp, m_op_flags, &this->trace);
```

Comment:
The trace is [now](https://github.com/ceph/ceph/pull/11053/commits/1578b3b7a6e10093bb0ffe0c470c713ff0ce1454) passed to the AioObjectRequest